docs/develop/guides/working-with-time-series-data.md

```diff
@@ -0,0 +1,63 @@
+---
+sidebar_position: 5
+sidebar_label: Working with Time Series Data
+title: Working with Time Series Data
+description: Learn how to store time series data in Momento Cache
+pagination_next: null
+---
+
+## Storing Time Series Data
+
+Temporarily storing time series data enables you to visualize that data without having to worry about longterm storage costs. Let's think about a common use case for time series data like IoT sensors. Sure, it's useful to see what the recent data looks like, but do you need to keep each sensor value indefinitely? With numerous sensors sending metrics into your database, your storage costs will skyrocket. Storing that data in Momento instead is a great way to take advantage of what Momento does best - storing large volumes of data without having to worry about scaling up or longterm storage costs.
+
+You can store time series data with the SortedSet data type. Using SortedSet automatically sorts your data by timestamp, making it easy to visualize that data using charts and graphs. You can take advantage of multiple access patterns using the Momento SDK's SortedSetFetchByRank and SortedSetFetchByScore. Using those methods enables you to retrieve a certain number of datapoints or datapoints for a certain time range.
```

Comment:
You can or you should? Perhaps explain with a sentence or two what would be best practice for time series data and why?

Reply:
Addressed in updated version